### DISCUSSION TITLE
New Built-In Pie Chart: Ability to Specify Legend Values Header

### DISCUSSION THREAD

**Discussion by societymartingale** [grafana/grafana#46066] [Dashboards, Panels & Visualizations]
In the old pie chart, there was an option to specify the values header in the legend. I am not seeing a way to do this in the new, built-in pie chart. Is this something that could be added if it doesn't already exist?

Here is the panel config for the old pie chart. 
![Screen Shot 2022-03-01 at 4 58 06 PM](https://user-images.githubusercontent.com/22607854/156256622-a1c4f416-550c-498b-b610-534e7a2b2c80.png)

Oftentimes, we like to set the header to something such as "Count", so it doesn't appear as simply "Value":

![Screen Shot 2022-03-01 at 5 08 26 PM](https://user-images.githubusercontent.com/22607854/156257088-d9c6efc8-db17-42d7-b8f2-2ba5e9fb1150.png)

(Someone also posted this request to the community channel at https://community.grafana.com/t/issues-with-new-pie-chart/51915/2 .)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
